### DISCUSSION TITLE
What is "unbound" in the `brew services list`?

### DISCUSSION THREAD

**Discussion by veganstraightedge** [Homebrew/discussions#129] [Everyday usage] [answered]
My MacBook battery died, computer shut down.
When I powered back up, Postgresql was dead in the water.

```sh
>  brew services list

Name       Status  User Plist
postgresql error   s    /Users/s/Library/LaunchAgents/homebrew.mxcl.postgresql.plist
redis      started s    /Users/s/Library/LaunchAgents/homebrew.mxcl.redis.plist
unbound    stopped      
```

Stopping/starting, restarting, uninstalling/re-installing, restarting my computer, deleting `postmaster.pid`… nothing works.

The one clue that I can't find an answer for is that `unbound stopped` item listed in the services. 

What is that? 
How do I make it go away?

**Comment by jonchang** [comment, marked as answer]
See `brew info unbound`. Likely it was installed as a dependency of something else you needed.

**Comment by veganstraightedge** [comment]
Oh jeez. You're right. I was reading it as "there is a service which is somehow become 'unbound'". Whatever that would even mean. Thanks for pointing out what was right in front of me! 

`¯\_(ツ)_/¯` 

```sh
> brew info unbound

unbound: stable 1.12.0 (bottled), HEAD
Validating, recursive, caching DNS resolver
https://www.unbound.net
/usr/local/Cellar/unbound/1.12.0 (57 files, 5.4MB) *
  Poured from bottle on 2020-11-08 at 16:09:40
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/unbound.rb
License: BSD-3-Clause
==> Dependencies
Required: libevent ✔, openssl@1.1 ✔
==> Options
--HEAD
	Install HEAD version
==> Caveats
To have launchd start unbound now and restart at startup:
  sudo brew services start unbound
==> Analytics
install: 247,980 (30 days), 607,380 (90 days), 2,073,225 (365 days)
install-on-request: 38,219 (30 days), 54,607 (90 days), 86,480 (365 days)
build-error: 0 (30 days)
```

**Reply by ddh** [reply]
Came here to post I was thinking the same exact thing 😆

**Reply by kagarmoe** [reply]
Yep. Same thing.

**Reply by beesanderson** [reply]
so do we reinstall it?

**Reply by GnaixEuy** [reply]
So, how to solve it？

**Reply by carlocab** [reply]
Solve what, exactly? There is nothing to solve if what you see is
```
unbound    stopped
```
in `brew services list`.

**Reply by GnaixEuy** [reply]
Emmmmm,I'm sorry I misunderstood the software before

**Comment by jaredatron** [comment]
https://unbound.docs.nlnetlabs.nl/en/latest/index.html

**Comment by rawnly** [comment]
```
Error: Refusing to uninstall /opt/homebrew/Cellar/unbound/1.17.1
because it is required by ffmpeg, gnupg, gnutls, task and vhs, which are currently installed.
You can override this and force removal with:
  brew uninstall --ignore-dependencies unbound
```

Hope this helps
